packages/vhd-lib/Vhd/VhdFile.integ.spec.js

```diff
@@ -42,6 +42,19 @@ afterEach(async () => {
   disposeHandler()
 })
 
+test('checkFile fails with unvalid VHD file', async () => {
+  const initalSize = 4
+  const rawFileName = `${tempDir}/randomfile`
+  await createRandomFile(rawFileName, initalSize)
+  const vhdFileName = `${tempDir}/vhdFile.vhd`
+  await convertFromRawToVhd(rawFileName, vhdFileName)
+
+  await checkFile(vhdFileName)
+
+  await fs.truncate(vhdFileName, 2)
+  await expect(async () => await checkFile(vhdFileName)).rejects.toThrow()
+})
```

Comment:
This is not a `VhdFile` related test, if you really to test this test utility, move it into a dedicated test file.